consul.json

```diff
@@ -0,0 +1,5 @@
+{
+ "advertise_addr": "127.0.0.1",
+ "data_dir": "/home/deep/consul_data",
+ "log_file": "/"
+}
```

Comment:
I think you may have accidentally committed this file. It is not used by any tests. You can
 `git rm --cached consul.json` to remove it from the git index without removing the file.